app/models/responses/SCTYCase.scala

```diff
@@ -27,14 +27,15 @@ case class SCTYCase(
                      procedureCode: String,
                      caseStatus: String,
                      caseSubStatus: Option[String],
+                     caseType: Option[CaseType],
                      goods: Option[Seq[Goods]],
                      declarantEORI: String,
                      importerEORI: String,
                      claimantEORI: Option[String],
                      totalCustomsClaimAmount: Option[String],
                      totalVATClaimAmount: Option[String],
                      totalClaimAmount: Option[String],
-                     totalReimbursmentAmount: Option[String],
+                     totalReimbursementAmount: Option[String],
```

Comment:
This would need a change on both sides: here and in the cds-reimbursement-claim

Reply:
I've checked in cds-reimbursement-claim and it is spelled totalReimbursementAmount there, I think it was just a spelling mistake in this repo